patches/@orama+plugin-docusaurus-v3+3.1.16.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js b/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js
+index eab29fc..2555aa0 100644
+--- a/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js
++++ b/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/index.js
```

Comment:
We should open a PR on their repo instead:
https://github.com/oramasearch/orama/tree/main/packages/plugin-docusaurus-v3

Reply:
Totally agree with this.
I'll open a PR to Orama to make it happen.